lib/releases.js

```diff
@@ -237,17 +237,36 @@ const generateChangeLog = (mergedPullRequests, config) => {
     changeLog.push(pullRequestToString(uncategorizedPullRequests), '\n\n')
   }
 
-  categorizedPullRequests.map((category, index) => {
-    if (category.pullRequests.length > 0) {
+  for (const [index, category] of categorizedPullRequests.entries()) {
+    if (category.pullRequests.length === 0) {
+      continue
+    }
+    const categoryTitle = template(config['category-template'], {
+      $TITLE: category.title,
+    })
+    const pullRequestString = pullRequestToString(category.pullRequests)
+
+    if (
+      !category.collapse ||
+      (category.collapse && category.pullRequests.length < 4)
```

Comment:
If you think this needs to be configurable, feel free to add it :)

Please follow the current config naming with `word-another-word`

Reply:
We could, but I have no idea what to name it 🙈  Any suggestions?

Reply:
`collapse-at` is fine it will go together with `collapse`. The readme would have to document whether it is greater than or less than that is used.
The default may be greater than 3?